features/Jisc/authenticate-storage-service-user.feature

```diff
@@ -0,0 +1,71 @@
+Feature: Authenticate Storage Service User
+  As an (Archivematica) Storage Service User, I want to login using my existing username and password, 
+  so that I can access the dashboard without needing to remember another password.
+  
+  This features uses the term "Identity Provider" to refer to the instition 
+  that provides a user with their account and authentication of their identity.
+  
+Scenario: Preservation-User logs in with correct credentials
+  Given <user> has an existing account with her identity provider 
+    And <user> has the entitlement <entitlement>
+    And <user> has an existing Storage Service user account 
+    And <user> is not already logged in to Storage Service
+  
+  When <user> enters (or clicks) a URL for the Archivematica dashboard
+    And Storage Service determines <user> is not already logged in
+    And Storage Service redirects <user> to the Identity Provider
+    And the Identity Provider determines the <user> does not have an existing authenticated session
+  
+  Then the Identity Provider presents a login page
+    And <user> enters their user name <username> and password <password>
+    And the Identity Provider authenticates <user>
+    And the Identity Provider presents an Information Release consent page
+    And <user> selects the option "Ask me again at next login" and clicks "accept"
+    And the Identity Provider redirects <user> to Storage Service
+    And Storage Service validates the response from the Identity Provider
+    And <user> is logged in with <role> privileges
+    And <user> is presented with the default transfer page 
```

Comment:
Rather than "default transfer page" I think for storage service this should be something like "storage service homepage"